rs/nns/governance/canister/governance.did

```diff
@@ -577,6 +626,62 @@ type Neuron = record {
   transfer : opt NeuronStakeTransfer;
   known_neuron_data : opt KnownNeuronData;
   spawn_at_timestamp_seconds : opt nat64;
+
+  // When a neuron is first created, this is set to create_timestamp_seconds.
+  //
+  // This is part of the "periodic confirmation" feature, which was proposed and
+  // (overwhelmingly) adopted in NNS motion proposal 132411:
+  // https://dashboard.internetcomputer.org/proposal/132411
+  //
+  // This gets updated to the current time when the user performs related
+  // actions, as described in the aforementioned proposal.
+  //
+  // For neurons created prior to this feature, this field will be set to
+  // 2024-09-01T00:00:00 UTC.
+  //
+  // Per NNS policy, this is opt. Nevertheless, it will never be null (unless we
+  // later deprecate this feature).
+  voting_power_refreshed_timestamp_seconds : opt nat64;
+
+  // The amount of "sway" this neuron has when voting on proposals.
+  //
+  // If a neuron regularly refreshes its voting power, this value is the same as
+  // potential_voting_power.
+  //
+  // However, if a neuron has not refreshed in more than six months, this will
+  // be less than potential voting power.
+  //
+  // As a further secondary consequence, the neuron receives less voting rewards
+  // (when it votes indirectly via following).
+  //
+  // For details, see https://dashboard.internetcomputer.org/proposal/132411.
+  //
+  // Per NNS policy, this is opt. Nevertheless, it will never be null (unless we
+  // later deprecate this feature).
```

Comment:
```suggestion
  // The amount of "sway" this neuron has when voting on proposals.
  //
  // If a neuron regularly votes, sets following or confirms following, this value is the same as
  // potential_voting_power.
  //
  // However, if a neuron has not performed any of these actions in in more than start_reducing_voting_power_after_seconds, this will
  // be less than potential voting power.
  //
  // As a further secondary consequence, the neuron receives less voting rewards.
  //
  // Per NNS policy, this is opt. Nevertheless, it will never be null.
```

Made concrete suggestions, hope the main motivation is clear from comments above.

Reply:
I think it's not good to list what triggers a refresh **inline**, because a) I would not assume that this set of actions is stable, b) it interrupts the flow. What I would do instead is later/separately list the actions that cause a refresh.

Reply:
OK sounds good, but I don't see where this is listed later. Do you also refer to the comment here?

Reply:
It isn't. I meant I would add that.

Reply:
Done.